### ISSUE TITLE
Support for USBHost

### ISSUE THREAD

**Issue by TheWanderer1983** [tsowell/vtbt#3]
Hello again,

I noticed that you are using the ESP32-C3-DevKitM-1. It has support for the usbhost. See an example https://hackaday.com/2024/11/17/esp32-hosts-a-usb-keyboard-in-this-typewriter/
and
https://github.com/tanakamasayuki/EspUsbHost

It would be great to be able to use a wired keyboard not just bluetooth.

Regards
Dylan

**Comment by tsowell** [comment]
I agree that would be nice - most of my keyboards are wired, after all. I may get it to it some day, but for now I don't have a specific need for it.
